modules/learn/pages/data/transactions.adoc

```diff
@@ -0,0 +1,56 @@
+= Distributed ACID Transactions
+
+[abstract]
+_Distributed ACID Transactions_ guarantee that multiple documents can be updated atomically.
+
+[#transactions-and-couchbase-server]
+== Transactions and Couchbase Server
+
```

Comment:
I feel we've lost the 'sales pitch' of the current docs here; we're straight into dry technical details without selling the benefits of our transactions.

Reply:
Right. Well, it has been a strong principle that in this section of the docs, we don't do sales or marketing. We've worked hard to keep it to the technical details. That's the function of the section. And that's also the issue with addressing this content here.